docs/specs/priority-queue.md

```diff
@@ -77,9 +79,9 @@ public class PriorityQueue<T> : IQueue<T>
 }
 ```
 
-### Scenarios
+### Examples
 
-#### (2)
+#### (1)
```

Comment:
Similar to this change (changing heading from Scenario to Examples and updating the numbers), consider changing it here too: https://github.com/TimLovellSmith/corefxlab/blob/d61eda595a030aee4f0563bebb400052d8ab0c13/docs/specs/priority-queue.md#scenario